lang/cs/Org.Apache.REEF.IMRU.Examples/PipelinedBroadcastReduce/FaultTolerantPipelinedBroadcastAndReduce.cs

```diff
@@ -85,35 +111,54 @@ internal static bool IsEvaluatorFailure(int failureType)
             }
         }
 
+        [NamedParameter(Documentation = "Total number of failures in recovery.", DefaultValue = "2")]
+        internal class TotalNumberOfAllowedFailures : Name<int>
```

Comment:
Maybe a better name would be `TotalNumberOfForcedFailures`? It's not that we _allow_ that many failures, it's that we fail intentionally that many times